### DISCUSSION TITLE
I see thousands of very small (0.000000000002xch) incoming transactions to my wallet every 2 minutes

### DISCUSSION THREAD

**Discussion by santivu** [Chia-Network/chia-blockchain#11303] [Support]
Since a few weeks, I see this into my wallet, producing a very slow synchronization:

https://www.chiaexplorer.com/blockchain/address/xch1sex3wyax4cv7nkdqtkxcxfe7xd9xujsrk23wmhtzcv03pldst5aqxw5d8f
![Captura de Pantalla 2022-04-25 a las 15 06 49](https://user-images.githubusercontent.com/88079415/165095152-65b81e95-2072-4476-bede-4ecfbc732205.png)

**Comment by loppefaaret** [comment]
yeah, someone is "dusting" the addresses seen on the blockchain, and there isn't much you can do about it.

**Comment by santivu** [comment]
I see.
Could potentially affect to win a block (or see it)? Because I wallet is almost freeze and impossible to make a transaction.

Would be nice to have a mechanism to block this kind of very small quantities in every wallet.

Thanks,



-Santi

**Reply by loppefaaret** [reply]
we are working on some possible solution to the problem... but it is not quite as easy as one might think

**Reply by santivu** [reply]
I can imagine. Also is not possible to ban at least temporally those addresses?

Do you know if could affect the winning of a block?
Many thanks for your support,

**Reply by loppefaaret** [reply]
As long as the wallet has passed the last plotNFT edit, it should not have any effect on farming

**Comment by santivu** [comment]
I can confirm to you that unfortunately is affecting my farming.



-Santi

**Reply by loppefaaret** [reply]
what is going on ? how is it having an effect on your farm ? the farming process is depending on the node, farmer and harvester process, as long as the wallet is synced passed the last plotNFT edit, and populated the config with the correct settings, the wallet process is not involved in farming ?

**Comment by NTOHPN** [comment]
xch1pqk4wtz8uhjf2yfqur6fqzls28q9xpjrlqrm8r2fw9jme83uzqyqpvq7pl请让它进入我的钱包

**Comment by santivu** [comment]
The stupidity of the day.



-Santi

**Comment by santivu** [comment]
Well, that is the reason I was contacting you, as Support. I have more than 750TiB farming, since July last year, growing from 150TiB. And I was winning consistenly a block under normal stimation time, getting 18 blocks won. Now with the upgrade to 1.3.0 and now 1.3.4, all was a mess, having to recreate the wallet until 3 times from scrath, as getting a wrong pool addres I was using last year and not getting properly at the end the solo status.Also the two addresses appears in the client with a waring, I fixed recovering an old config.yaml.With this “personal storm” to my wallet, you can imagine how sloooooow is the syncronization process at the end and the loose of confidence in the farming process, now with more than 43 days without wining any block (my client says 8 days as estimation). Have a nice evening,  -Santi De: loppefaaretEnviado: lunes, 2 de mayo de 2022 16:15Para: Chia-Network/chia-blockchainCC: santivu; AuthorAsunto: Re: [Chia-Network/chia-blockchain] I see thousands of very small (0.000000000002xch) incoming transactions to my wallet every 2 minutes (Discussion #11303) what is going on ? how is it having an effect on your farm ? the farming process is depending on the node, farmer and harvester process, as long as the wallet is synced passed the last plotNFT edit, and populated the config with the correct settings, the wallet process is not involved in farming ?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by santivu** [comment]
Hi, 

To continue giving additional information, this “personal” storm on my wallet is causing things like this.
I’m trying to move my full node to my main computer, an Intel i9-10900K with 64Gb RAM and 2x 2TB nvme Samsung 980 Pro as storage, using a backup of db and wallet from yesterday afternoon from my current full node. The blockchain db has been syncronized in almost 10 minutes, all fine with that.

But after 6 hours, the wallet is still on this state:  

![1](https://user-images.githubusercontent.com/88079415/168682252-876992b1-8624-4537-8641-0ed457ef4f84.png)

And  crashing. In the GUI I see this:

![2](https://user-images.githubusercontent.com/88079415/168682350-704f3048-dcfa-456b-8164-e3b46cb05577.png)

Look at the number of pages.
Please help with this nightmare. Thanks,  

-Santi
